Sources/ClusteringManager.swift

```diff
@@ -135,7 +136,8 @@ public final class ClusteringManager {
   // Add only the annotations we need in the current region
   // https://robots.thoughtbot.com/how-to-handle-large-amounts-of-data-on-maps#adding-only-the-annotations-we-need
   private func reload(annotations: [MKAnnotation], onMapView mapView: MKMapView, completion: Completion?) {
-    let currentSet = NSMutableSet(array: mapView.annotations)
+    let mapAnnotations = mapView.annotations.filter(filterAnnotations)
+    let currentSet = NSMutableSet(array: mapAnnotations)
```

Comment:
@onmyway133 I had troubles with `Set`, because `MKAnnotation` is protocol and not `Hashable`. It's possible do `Set(mapAnnotations as! [NSObject])`, but I don't like unwrapping.